### PR TITLE
Moving commands from core

### PR DESCRIPTION
**Pull request by tonibm19**
Moved:
/fly
/locate
/back
/tpa and related
/top
/vanish
/tps

### CODE REVIEW

**Comment by mathias-github** [comment]
Aren't es.getpos and es.locate basically duplicates? Maybe add GetWorld to es.getpos and remove es.locate?

**Comment by tonibm19** [comment]
But I think some of those commands are essential (or nearly) for the server
What do the others thing?

**Comment by NiLSPACE** [comment]
Nah, I think those commands can go into essentials. Except maybe /regen, that could go into ChunkWorx.

**Comment by tonibm19** [comment]
I think its ready, what do you think?

Ill start with the Core removing part

**Comment by NiLSPACE** [comment]
I believe you can remove all the places where SetBackCoordinates is used, and simply use the OnEntityTeleport hook. You can even make an array out of it so more coordinates can be saved.

**Comment by tonibm19** [comment]
Like this?

**Comment by NiLSPACE** [comment]
Yup ^^

**Comment by tonibm19** [comment]
So?

**Comment by tonibm19** [comment]
Now I understood why cIniFile must be a local variable D:

**Comment by tonibm19** [comment]
Fixed

**Comment by tonibm19** [comment]
Ini seems fixed in all the plugin
Now I have this error
[20:56:57] LUA: Plugins/Essentials/jails.lua:24: attempt to index global 'Jailed
' (a boolean value)
[20:56:57] Stack trace:
[20:56:57]   Plugins/Essentials/jails.lua(24): (no name)
[20:56:57]   [C](-1): FindAndDoWithPlayer
[20:56:57]   Plugins/Essentials/jails.lua(30): (no name)
[20:56:57] Stack trace end
[20:56:57] Error in <attached> calling function <callback>()
[20:57:04] LUA: Plugins/Essentials/hooks.lua:90: attempt to index global 'Jailed
' (a boolean value)
[20:57:04] Stack trace:
[20:57:04]   Plugins/Essentials/hooks.lua(90): (no name)
[20:57:04] Stack trace end
[20:57:04] Error in plugin Essentials calling function <callback>()

why?

File: jails.lua [hunk 1]
```diff
@@ -23,9 +19,11 @@ function HandleJailCommand( Split, Player )
 			end
 			OtherPlayer:TeleportToCoords( jails[Tag]["x"] + 0.5 , jails[Tag]["y"] , jails[Tag]["z"] + 0.5)
 			OtherPlayer:SendMessageWarning('You have been jailed')
-			UsersIni:SetValue(OtherPlayer:GetName(),   "Jailed",   "true")
-			UsersIni:WriteFile("users.ini")
+			UsersINI:SetValue(OtherPlayer:GetName(),   "Jailed",   "true")
+			UsersINI:WriteFile("users.ini")
+			Jailed[OtherPlayer:GetName()] = true
 			Jailed = true
```
Comment (NiLSPACE) [inline jails.lua]: I believe this is the problem.

Comment (tonibm19) [inline jails.lua]: But one is an array and the other a server variable.
Didnt know they could collide, but thanks, ill try later.

**Comment by tonibm19** [comment]
Well, I think its ready now.
If there are now objections I'm going to merge this and Core PR tomorrow morning (EU Time)
You can merge it before if you consider its ready.

**Comment by tonibm19** [comment]
Done

**Comment by NiLSPACE** [comment]
Great, I'd say merge :)

**Comment by tonibm19** [comment]
Fixed an issue where massive jail escape could be possible.

I will merge this afternon then, if there are no objections.

**Pull request merged**